keyResolver.js

```diff
@@ -17,16 +17,10 @@ async function keyResolver({id} = {}) {
   if(id.startsWith('did:')) {
     return didIo.get({did: id, forceConstruct: true});
   }
-  if(id.startsWith(location.origin)) {
-    const headers = {Accept: 'application/ld+json, application/json'};
-    const response = await httpClient.get(id, {
-      headers
-    });
-    return response.data;
-  }
-  throw new Error(
-    `"id" must start with either "did:" or "${location.origin}".`
-  );
+
+  const response = await httpClient.get(id);
```

Comment:
I *think* this is ok? The only other option would be to lock it down to a list of KMS URLs obtained from the server. I'm not sure if that is really needed as a security measure here though. I guess we can proceed with this.